package.json

```diff
@@ -120,26 +125,38 @@
     "prettier": "1.18.2",
     "react-test-renderer": "^16.8.4",
     "react-testing-library": "^7.0.0",
+    "rehype": "^7.0.0",
     "rimraf": "^3.0.0",
     "simple-git": "^1.124.0",
     "start-server-and-test": "^1.7.11",
+    "style-loader": "^0.23.1",
     "stylelint": "^9.10.1",
     "stylelint-config-recommended": "^2.1.0",
     "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-processor-styled-components": "^1.5.2",
     "svg-inline-loader": "^0.8.0",
     "to-string-loader": "^1.1.5",
+    "unist-util-visit": "^1.4.0",
     "webpack": "^4.29.6",
     "webpack-cli": "^3.2.3",
     "webpack-dev-server": "^3.2.1"
   },
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "husky",
+      "run-node"
```

Comment:
Best practice stuff for Husky w/ Lerna, attempting to make it work with Git worktrees, no luck so far.